src/main/java/com/varukha/onlinebookstore/dto/cartItem/UpdateCartItemRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package com.varukha.onlinebookstore.dto.cartitem;
+
+import jakarta.validation.constraints.Min;
+import lombok.Data;
+
+@Data
+public class UpdateCartItemRequestDto {
+    @Min(0)
+    private int quantity;
```

Comment:
Not null?

Reply:
Changed to Integer and added @NotNull.

Reply:
On this stage i really not sure what better to use, primitive or object type.